app/routes/about.tsx

```diff
@@ -58,6 +58,9 @@ export default function About() {
 				<li>
 					<strong>Xavier Ruiz</strong>
 				</li>
+				<li>
+					<strong>Sean Sanker Jr</strong>
```

Comment:
Could you add `"Sanker"` at https://github.com/SquiggleTools/boston-ts-website/blob/main/cspell.json#L24? 🙈